src/components/Dao/DaoSidebar.tsx

```diff
@@ -241,4 +253,32 @@ const SubscribedTokenBalance = withSubscription({
   },
 });
 
-export default connect(mapStateToProps)(DaoSidebar);
+const SubscribedDaoSidebar = withSubscription({
```

Comment:
intriguing use of withSubscriptions. seemed weird at first, but I guess it makes some sense